codegen/facelift/templates/IPCServiceAdapter.template.h

```diff
@@ -83,7 +82,13 @@ class {{classExport}} {{className}}: public {{baseClass}}
 
     void connectSignals() override;
 
-    void serializePropertyValues(OutputIPCMessage& msg, bool isCompleteSnapshot) override;
+    QVariantMap changedProperties();
```

Comment:
changedProperties vs marshalProperties ??

Reply:
what do you mean? these two functions have completely different purposes.